packages/interfaces/src/message.ts

```diff
@@ -2,7 +2,10 @@ import type { PubsubTopic } from "./misc.js";
 
 export interface SingleShardInfo {
   clusterId: number;
-  shard: number;
+  /**
+   * Specifying this field indicates to the encoder/decoder that static sharding should be used.
```

Comment:
```suggestion
   * Specifying this field indicates to the encoder/decoder that static sharding must be used.
```
Just to clarify is not a preference.

Reply:
fixed